sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -6,15 +6,506 @@
 
 from ...fields import consts
 from ..xcvr_api import XcvrApi
-
+import time
+BYTELENGTH = 8
 class CmisApi(XcvrApi):
     NUM_CHANNELS = 8
 
     def __init__(self, xcvr_eeprom):
         super(CmisApi, self).__init__(xcvr_eeprom)
 
+    # Transceiver Information
     def get_model(self):
         return self.xcvr_eeprom.read(consts.VENDOR_PART_NO_FIELD)
 
+    def get_vendor_rev(self):
```

Comment:
Need api to know CMIS revision (Page0, Byte1)
Need api to know Identifier (Page0, Byte0)

Reply:
Function get_module_type (Page0, Byte128) return the identifier as well (same as Page0, Byte0).
Function get_cmis_rev (Page0, Byte1) returns the CMIS revision.